misk/src/main/kotlin/misk/web/interceptors/MetricsInterceptor.kt

```diff
@@ -0,0 +1,30 @@
+package misk.web.interceptors
+
+import misk.Action
+import misk.Chain
+import misk.Interceptor
+import misk.metrics.Metrics
+import misk.metrics.MetricsScope
+import misk.web.Response
+import javax.inject.Inject
+import javax.inject.Singleton
+
+internal class MetricsInterceptor internal constructor(val scope: MetricsScope) : Interceptor {
+
+  @Singleton
+  class Factory @Inject constructor(val m: Metrics) : Interceptor.Factory {
```

Comment:
In general I'm opposed to single letter variable names, though this is such a small usage it's not a big deal. My vote would be `m` => `metrics` though.